pipelines/test/pkgconf.yaml

```diff
@@ -0,0 +1,21 @@
+name: pkgconf
+
+needs:
+  packages:
+    - pkgconf
+
+pipeline:
+  - name: pkgconf build dependency check
+    runs: |
+      # Get our dev package name, which is possibly a sub package
+      dev_pkg=$(basename ${{targets.contextdir}})
+      # Find all pc files installed by this dev_pkg
+      for pc_file in $(apk info -L "$dev_pkg" | grep "\.pc$"); do
+        # Try to get the library name from the pc filename
+        lib_name=$(basename "$pc_file" ".pc")
+        # Run some package config tests, which will likely be used by
+        # other packages that have build dependencies on this dev package
+        pkgconf --modversion $lib_name
+        pkgconf --libs $lib_name
+        #pkgconf --cflags $lib_name
```

Comment:
Is this a leftover? can be removed?

Reply:
or uncommented

Reply:
Maybe, not sure.  @xnox previously thought that --cflags should succeed everywhere but my testing is that it doesn't necessarily.  I'll remove it for cleaner code and just take a mental note to maybe readd it later.